### ISSUE TITLE
Missing documentation for "services" property on "google_project"

### ISSUE THREAD

**Issue by zbjornson** [hashicorp/terraform#14452]
### Affected Resource(s)
- google_project

The example usage on this page has a `services` array that I don't see documented (unless it's some generic property applicable to any Terraform resource that I'm not aware of). It looks like the Google APIs that are enabled for that project, but the example values are not ones I can find by Googling either.

https://www.terraform.io/docs/providers/google/r/google_project.html

**Comment by zbjornson** [comment]
I see this now on https://www.terraform.io/docs/providers/google/r/google_project_services.html, but it should probably be repeated or linked from the google_project page.

**Comment by paddycarver** [comment]
Hi @zbjornson!

From my investigation, it looks like the `services` field of the `google_project` resource was never released. If it was released, it has been deprecated and removed, but I can't find any trace of it.

My guess is the docs just weren't updated as we iterated on the PR for the original resource. I've got a PR ready to remove mention of `services` from the `google_project` page, which I'll open in the morning when my internet (hopefully!) is functioning again.

Thanks for bringing this to our attention! Sorry for the confusion.
